platform/landing-page/pom.xml

```diff
@@ -50,7 +50,7 @@
         <dependency>
             <groupId>org.apache.servicemix.specs</groupId>
             <artifactId>org.apache.servicemix.specs.jsr339-api-2.0</artifactId>
-            <version>2.4.0</version>
+            <version>2.5.0</version>
```

Comment:
Instead of hard-coding the version here, could we move the `cxf.servicemix.specs.version` property defined in platform-app's pom.xml to the platform's pom.xml and use that here?